UnitySetup/UnitySetup.psm1

```diff
@@ -1870,7 +1870,36 @@ function Start-UnityEditor {
             $process.Start() | Out-Null
 
             if ( $Wait ) {
-                $process.WaitForExit()
+                if( $LogFile ) {
+                    $ljob = Start-Job -ScriptBlock {
+                        param($log)
+
+                        while ( -not (Test-Path $log -Type Leaf) ) {
+                            Start-Sleep -Milliseconds 100
+                        }
+
+                        Get-Content "$log" -Wait
```

Comment:
```suggestion
                        Get-Content "$log" -Wait  | Write-Host
```

Reply:
I'm using `-PassThru` anyway, not sure if that'll change anything.

Reply:
I don't think you want the Write-Host in the job.